src/services/leafletHelpers.js

```diff
@@ -658,6 +658,11 @@ angular.module('ui-leaflet').service('leafletHelpers', function ($q, $log) {
                 }
             }
```

Comment:
BTW we are moving to a more plugin/micro services based architecture. So this will eventually need to be setup at config time to extend this service via a decorator.